4-WebApp-your-API/4-1-MyOrg/AppCreationScripts-withCert/sample.json

```diff
@@ -94,8 +94,19 @@
         "SampleSubPath": "4-WebApp-Your-API\\4-1-MyOrg",
         "ProjectDirectory": "\\TodoListService"
       },
-      "AppScopes": [ "ToDoList.Read", "ToDoList.Write" ]
-
+      "Scopes": [ "ToDoList.Read", "ToDoList.ReadWrite" ],
+      "AppRoles": [
```

Comment:
@kalyankrishna1 : didn't you want to use the term AppPermissions?

Reply:
@jmprieur we are using AppRoles instead of AppScopes now. Inside Code Gen we convert Application Permissions to AppPermissions and User permissions to AppRoles. 
@kalyankrishna1 - I think we can chat about a better term for sample.json?

Reply:
@aremo-ms : that's fine. It's not totally customer facing so no issues.

Reply:
it actually gets filtered and placed into a property named "AppPermissions" in the code gen JM. 
We can simplify this, but the issue with moving away from how its organized in manifest is that over a period of time requirements will force you kind of match everything that's in the manifest anyways.